exasol/analytics/audit/columns.py

```diff
@@ -0,0 +1,61 @@
+from exasol.analytics.schema import (
+    decimal_column,
+    timestamp_column,
+    varchar_column,
+)
+
+
+class BaseAuditColumns:
```

Comment:
Actually, I am not yet convinced by the proposal.
Are there also non-base-audit columns?

Reply:
Is clarified. I will keep `BaseAuditColumns`.

Reply:
yes, there are, especially columns defined by other extensions

Reply:
see next push, keeping the name `BaseAuditColumns`